package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scripts": {
+    "test": "cypress run",
+    "test:open": "cypress open"
+  },
+  "dependencies": {
+    "cypress": "^9.6.1"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
+    "@typescript-eslint/parser": "^5.23.0",
+    "eslint": "^8.15.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "typescript": "^4.6.4"
+  }
+}
```

Comment:
Did you initialized the project with the steps specified in section 2 of workshop? I don't see the author and other properties in this file.
https://github.com/testing-community/cypress-training-doc#2-configuracion-inicial-del-proyecto